### PR TITLE
ArchLinux ARM

### PR DESCRIPTION
**Pull request by petrbel**

### CODE REVIEW

File: dockerfiles/archlinuxarm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+FROM scratch
+LABEL thanksto "archimg/archlinux"
+LABEL maintainer "Iterait a.s. <docker@iterait.com>"
+
+# add image content
+ADD arch-rootfs/ /
+
+# add
```
Comment (FloopCZ) [inline dockerfiles/archlinuxarm/Dockerfile]:
```suggestion
```

File: dockerfiles/archlinuxarm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        xfsprogs \
+    && rm -rf \
+        /usr/share/man/* \
+        /var/cache/pacman/pkg/* \
+        /var/lib/pacman/sync/* \
+        /README \
+        /etc/pacman.d/mirrorlist.pacnew
+
+CMD ["/bin/bash"]
+
```
Comment (FloopCZ) [inline dockerfiles/archlinuxarm/Dockerfile]:
```suggestion
```

**Review by FloopCZ — APPROVED** [review]
Very nice, thank you @petrbel !

**Comment by blazekadam** [comment]
Thank you guys @petrbel @FloopCZ for pushing this forward.

**Pull request merged**